### PR TITLE
Set release date for 0.11.3 in notes

### PR DESCRIPTION
**Pull request by MrCreosote**

### CODE REVIEW

File: src/us/kbase/common/test/service/ServiceCheckerTest.java [hunk 1]
```diff
@@ -54,7 +54,7 @@ public static void beforeClass() throws Exception {
 		TESTS.put("short200", new TestSpec(
 				"URL " + KBASE_ENV + "/services/shock-api does not point " +
 				"to a KBase SDK generated service. Code: 200, message: OK, " +
-				"content: {\"attribute_indexes\"",
+				"content: {",
```
Comment (kkellerlbl) [inline src/us/kbase/common/test/service/ServiceCheckerTest.java:57]: Is this diff important?

**Review by kkellerlbl — APPROVED** [review]
Approved modulo one comment (which looks harmless).

File: src/us/kbase/common/test/service/ServiceCheckerTest.java (hunk 1 above)
Comment (MrCreosote) [reply src/us/kbase/common/test/service/ServiceCheckerTest.java:57]: It allows the tests to pass. `attribute_indexes` is shock only, the blobstore doesn't have that field.

Comment (kkellerlbl) [reply src/us/kbase/common/test/service/ServiceCheckerTest.java:57]: :+1:

**Pull request merged**